www/%team/distributing/index.html.spt

```diff
@@ -36,25 +36,25 @@ title = _("Team Members")
       </td>
       <td class="figure percentage"></td>
     </tr>
-    {% if user.participant == team %}
+    {% if user.participant.username == team.owner %}
     <tr>
       <td colspan="6" class="lookup">
-        <div id="lookup-container">
+        <div class="lookup-container">
           <form>
             <input name="csrf_token" type="hidden" value="{{ csrf_token }}" />
-            <input tabindex="1" id="query"
+            <input tabindex="1" class="query"
                    autocomplete="off"
                    placeholder="{{ _("Enter a Gratipay username") }}"
                />&nbsp;<button type="submit"
                                tabindex="2">{{ _("Add") }}</button>
-            <ul id="lookup-results"></ul>
+            <ul class="lookup-results"></ul>
           </form>
         </div>
       </td>
     </tr>
     {% endif %}
     </thead>
     <tfoot><tr><td colspan=6><div class="loading-indicator"/></td></tr></foot>
-    <tbody id="team-members"></tbody>
+    <tbody class="team-members"></tbody>
```

Comment:
I don't think it should be, and certainly not by default. Why _would_ we restrict it?